packages/flame_jenny/jenny/lib/src/dialogue_view.dart

```diff
@@ -27,7 +32,7 @@ import 'package:meta/meta.dart';
 /// be implemented either synchronously or asynchronously. In the latter case
 /// the dialogue runner will wait for the future to resolve before proceeding
 /// (futures from several dialogue views will be awaited simultaneously).
-abstract class DialogueView {
+mixin class DialogueView {
```

Comment:
I guess this should still be abstract since it isn't much use instantiating it?
```suggestion
abstract mixin class DialogueView {
```